etc/adminhtml/di.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <type name="Magento\Config\Model\Config\Structure\Element\Field">
-        <plugin name="scopehint_plugin_field" type="AvS\ScopeHint\Plugin\ConfigFieldPlugin" sortOrder="1" disabled="false" />
+        <plugin name="avstudnitz_scopehint_plugin_field" type="AvS\ScopeHint\Plugin\ConfigFieldPlugin" sortOrder="1" disabled="false" />
     </type>
-</config>
+
+    <type name="Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\Eav">
+        <plugin name="avstudnitz_scopehint_plguin_add_tooltip_to_product_field" type="AvS\ScopeHint\Plugin\ProductEavDataProviderPlugin" sortOrder="1" disabled="false" />
```

Comment:
Could you change the name to `avs_scopehint_plugin_add_tooltip_to_product_field`? (shorten my name and fix typo)